packages/langbase/src/langbase/langbase.ts

```diff
@@ -24,6 +24,29 @@ export interface RunOptionsStreamT extends RunOptionsBase {
 	stream: true;
 }
 
+export interface LlmOptionsBase {
+	messages: PromptMessage[];
+	model: string;
+	llmKey: string;
+	top_p?: number;
+	max_tokens?: number;
+	temperature?: number;
+	presence_penalty?: number;
+	frequency_penalty?: number;
+	stop?: string[];
+	tool_choice?: any;
```

Comment:
I think we have types for `tool_choice`.

Reply:
Updated them.